library/core/src/array/mod.rs

```diff
@@ -19,6 +19,20 @@ mod iter;
 #[unstable(feature = "array_value_iter", issue = "65798")]
 pub use iter::IntoIter;
 
+/// Converts a reference to `T` into a reference to an array of length 1 (without copying).
+#[unstable(feature = "array_from_ref", issue = "none")]
```

Comment:
Please file a tracking issue and replace `none` here and on `from_mut` below before merging.

Reply:
yeah, didn't want to open a tracking issue if we aren't yet sure about adding merging that PR.

Considering that you expressed some interest in adding these methods, I will just go ahead and do so now